mozilla/activity-stream
# Bug 1594537 - Make DS Responsive

Test steps:

- With DS enabled, resize the window and scroll around. Nothing should look broken.
- With DS disabled, resize the window and scroll around. Nothing should look broken.
